### DISCUSSION TITLE
How to prepare needed input files?

### DISCUSSION THREAD

**Discussion by sylenola** [team-ocean/veros#646] [Q&A]
What input data are must needed for a general circulation model (cold start) in Veros?

And what tools can I use to prepare these input files?

**Comment by dionhaefner** [comment]
I can't give specific guidance on this. Please look at the example setups, which include reading forcing + IC data.

**Reply by sylenola** [reply]
I found the acc example has no input files

**Reply by dionhaefner** [reply]
The realistic setups do (for example `global_4deg`)
